code-challenges.test.js

```diff
@@ -10,60 +18,153 @@
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+
+describe("wordsCapitalized", () => {
+   it("takes in an array of words and returns an array with all the words capitalized", () => {
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+     expect(wordsCapitalized(randomNouns1)).toEqual["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+     expect(wordsCapitalized(randomNouns2)).toEqual["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+   })
+ })
+// //ReferenceError: wordsCapitalized is not defined
+
+// //PsuedoCode:
+// //In order to get the first letter of each value in the array to be capitalized, the array needs to use .map to iterate over the array and return the same amount of values. Since it is just a single letter in each word, the index of each word, which would be zero, would need to be uppercase. 
+     //I had to do a lot of research to understand the example in the syllabus for higher order functions which is asking for the same output. I was not sure if there was a HOF which did just capitalization. After going through the resources provided, and google, all of the examples I saw were .toUpperCase the entire array, keeping zero index UpperCase and returning a .substring() method then joining the rest of the string together. I understand the concept but I had to look at the example for the syntax and the methods. 
+// //Declare a function: wordsCapitalized
+// //Parameters: randomNouns1, randomNouns2, 
+// //Each value of the array will have the zero index upper cased. value[0].toUpperCase which will capitalize the first letter. value.substring(1) will "concantenate the remainder of the word." Since the first letter was .toUpperCase, the rest of the string needs to be joined together using .join(). join() will bring them back into one string. 
+// //Return: "Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"
+         // "Temperature", "Database", "Chopsticks", "Mango", "Deduction"
+
+
+// // b) Create the function that makes the test pass.
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
-
 
-// b) Create the function that makes the test pass.
+ const wordsCapitalized = (array) => {
+    let newWords = array.map(value => {
+        return value[0].toUpperCase() + value.substring(1)
+    })
+        return newWords.join(" ")
```

Comment:
Not sure you would need join() since your expected output is an array.  Otherwise, excellent work on the logic and research!!